modules/eb_widget/src/Plugin/EntityBrowser/Widget/DropzoneJsEbWidget.php

```diff
@@ -211,4 +211,20 @@ protected function validateExtension($filename, $extensions) {
     return TRUE;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state) {
+    $configuration = $this->getConfiguration();
+    $settings = $configuration['settings'];
+    $form += parent::buildConfigurationForm($form, $form_state);
```

Comment:
Parent is WidgetBase, which doesn't do anything in buildConfigurationForm(). Only description is added here, but we have few more configuration options in defaultConfiguration(), which we would need to handle.